sdk/storage/azure-storage-file-share/out/azure/storage/fileshare/_generated/models/_azure_file_storage_enums.pyi

```diff
@@ -85,6 +89,21 @@ class CopyStatusType(str, Enum):
     aborted: str = ...
     failed: str = ...
 
+class LeaseDurationType(str, Enum):
+    infinite: str = ...
+    fixed: str = ...
+
+class LeaseStateType(str, Enum):
+    available: str = ...
+    leased: str = ...
+    expired: str = ...
+    breaking: str = ...
+    broken: str = ...
+
+class LeaseStatusType(str, Enum):
+    locked: str = ...
+    unlocked: str = ...
```

Comment:
If we expose Lease types, we should use a prefix to distinguish from Blobs, e.g. ShareLeaseDuration

Reply:
https://github.com/Azure/azure-sdk-for-python/pull/9759